14-truck_backer-upper.ipynb

```diff
@@ -243,16 +1290,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 10000/10000 [00:14<00:00, 677.17it/s]\n"
+     ]
+    }
+   ],
    "source": [
     "# Build expert data set\n",
     "\n",
     "episodes = 10\n",
     "inputs = list()\n",
     "outputs = list()\n",
-    "# truck = Truck(); episodes = 10_000  # uncooment for creating the data set\n",
```

Comment:
Don't change comments into executable instructions.

Reply:
I'm confused? The comment said  I should un-comment when creating data set